CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.5.1]
+
+### Fixed
+
+- Operator crash when creating Ingress without `spec.route` field defined
```

Comment:
Please add a link to this PR. 

Suggestions:
- Operator crash when creating Ingress without `spec.route` field defined --> Operator crashes when Ingress is created without defining `spec.route` field.
- Do not add `kubectl.kubernetes.io/last-applied-configuration` annotation to created resources to prevent unnecessary pod restarts --> Unnecessary pod restarts due to adding `kubectl.kubernetes.io/last-applied-configuration` to resources created by the operator.